src/main/java/seedu/todolist/logic/commands/AddCommand.java

```diff
@@ -0,0 +1,57 @@
+package seedu.todolist.logic.commands;
+
+import seedu.todolist.commons.exceptions.IllegalValueException;
+import seedu.todolist.model.tag.Tag;
+import seedu.todolist.model.tag.UniqueTagList;
+import seedu.todolist.model.task.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Adds a task to the to-do list.
+ */
+public class AddCommand extends Command {
+
+    public static final String COMMAND_WORD = "add";
```

Comment:
Is `public` the best access modifier for `COMMAND_WORD` ? As a rule of thumb start all data members as private, and depending on their usage promote them to protected, and if absolutely necessary, to public.